ChiTech/ChiPhysics/FieldFunction/lua/physics_exportfieldfunc.cc

```diff
@@ -78,7 +78,7 @@ int chiExportFieldFunctionToVTKG(lua_State *L)
     exit(EXIT_FAILURE);
   }
 
-  ff->ExportToVTKG(base_name,field_name);
+  ff->ExportToVTK(base_name, field_name);
```

Comment:
As the c++ method has been consolidated to ```ExportToVTK```, is it the case to make an analogous modification to the lua method(s) as well so that the scripting interface does not diverge (greatly) from the c++ interface?

Reply:
This is a very important observation. The original group-based thinking was responsible for this function having the appended G. Technically it would now be deprecated and has to go through a deprecation cycle, however, I am going to leave the scripting environment as-is for now (reason to follow).

Field functions need to undergo a major revamp as was indicated in the previous PR. Would you be okay with this remaining in a slightly diverged state till the revamp comes along?